services/core-api/app/api/verifiable_credentials/resources/verifiable_credential.py

```diff
@@ -0,0 +1,12 @@
+from flask_restplus import Resource
+
+from app.extensions import api
+
+from app.api.utils.resources_mixins import UserMixin
+from app.api.services.traction_service import TractionService
+
+class VerifiableCredentialResource(Resource, UserMixin):
+    @api.doc(description='', params={})
+    def get(self):
```

Comment:
Are you able to add a feature flag to these endpoints?

I added a flag named `verifiable_credentials` to our feature flag service.

How?
1. Register the flag in `feature_flag.py`
2. Use the `is_feature_enabled` check

Reply:
I added it to the namespace registration. i figured htat was better than each individual endpoint

Reply:
Good call!